Source/Scene/Cesium3DTileset.js

```diff
@@ -1510,13 +1509,31 @@ define([
         };
     }
 
+    function filterProcessingQueue(tileset) {
+        var tiles = tileset._processingQueue;
+        var length = tiles.length;
+
+        var removeCount = 0;
+        for (var i = 0; i < length; ++i) {
+            var tile = tiles[i];
+            if (tile._contentState !== Cesium3DTileContentState.PROCESSING) {
+                ++removeCount;
+                continue;
+            }
+            if (removeCount > 0) {
+                tiles[i - removeCount] = tile;
+            }
+        }
+        tiles.length -= removeCount;
```

Comment:
For whoever is reviewing, this code is based on a similar filter function in [RequestScheduler](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/RequestScheduler.js#L220-L243)